### DISCUSSION TITLE
Save My Cpp Intellisense

### DISCUSSION THREAD

**Discussion by kk9265** [microsoft/vscode-cpptools#12057] [Q&A]
In my project，one script will put multi cpp files into a big cpp file， then compile the big one。However this will leads the cpp  Intellisense not working. In addtional, i really don‘t want to modify the cpp files, because there are  so many, but if they compiled standalone, they just cannot be compiled.  for example,  a.cpp with include 'a.h' code and b.cpp, which  also need  the include code   but not, now  the script will put them into a new cpp file named c.cpp, then the compiler will just compile c.pp. Everything work fine, (because the  a.cpp has the include code). However， if just compile the b.cpp standalone,  we will get one compile error. Is there a viable solution? thx

**Comment by bobbrow** [comment]
Discussion was converted to issue.
